res/openvpn-gui-res-zh-hans.rc

```diff
@@ -415,12 +415,12 @@ BEGIN
 --passphrase_attempts\t: 允许尝试输入密码次数。\n\
 --connectscript_timeout\t: 等待连接脚本执行的时间。\n\
 --disconnectscript_timeout\t: 等待断开连接脚本执行的时间。\n\
---preconnectscript_timeout\t: Time to wait for preconnect script to finish.\n\
---iservice_admin\t\t: 0=Do not use interactive service when started as admin (default is 1 for Windows 7 and newer)\n\
```

Comment:
Consider changing "interactive service" to "Open VPN Interactive Service", as this is a program name not a generic word.